File: csharp/DVSClient/DVSClient/address/format/enrichment/GbrGovernment.cs
using DVSClient.Server.Address.Format;

namespace DVSClient.Address.Format.Enrichment
{
    public class GbrGovernment
    {
        public string EerCode { get; }
        public string EerCodePre2011 { get; }
        public string EerName { get; }
        public string GorCode { get; }
        public string GorCodePre2011 { get; }
        public string GorName { get; }
        public string LeaCode { get; }
        public string LeaName { get; }
        public string LaCode { get; }
        public string LaCodePre2011 { get; }
        public string LaName { get; }
        public string WardCode { get; }
        public string WardCodePre2011 { get; }
        public string WardName { get; }
        public string CensOutArea { get; }

        public GbrGovernment(RestApiEnrichmentGbrGovernment api)
        {
            EerCode = api.EerCode ?? string.Empty;
            EerCodePre2011 = api.EerCodePre2011 ?? string.Empty;
            EerName = api.EerName ?? string.Empty;
            GorCode = api.GorCode ?? string.Empty;
            GorCodePre2011 = api.GorCodePre2011 ?? string.Empty;
            GorName = api.GorName ?? string.Empty;
            LeaCode = api.LeaCode ?? string.Empty;
            LeaName = api.LeaName ?? string.Empty;
            LaCode = api.LaCode ?? string.Empty;
            LaCodePre2011 = api.LaCodePre2011 ?? string.Empty;
            LaName = api.LaName ?? string.Empty;
            WardCode = api.WardCode ?? string.Empty;
            WardCodePre2011 = api.WardCodePre2011 ?? string.Empty;
            WardName = api.WardName ?? string.Empty;
            CensOutArea = api.CensOutArea ?? string.Empty;
        }
    }
}

File: java/src/main/java/com/experian/dvs/client/address/format/enrichment/GbrGovernment.java
package com.experian.dvs.client.address.format.enrichment;

import com.experian.dvs.client.server.address.format.RestApiEnrichmentGbrGovernment;

import java.util.Objects;

public class GbrGovernment {

    private final String eerCode;
    private final String eerCodePre2011;
    private final String eerName;
    private final String gorCode;
    private final String gorCodePre2011;
    private final String gorName;
    private final String leaCode;
    private final String leaName;
    private final String laCode;
    private final String laCodePre2011;
    private final String laName;
    private final String wardCode;
    private final String wardCodePre2011;
    private final String wardName;
    private final String censOutArea;

    // Constructor
    public GbrGovernment(RestApiEnrichmentGbrGovernment api) {
        this.eerCode = Objects.toString(api.getEerCode(), "");
        this.eerCodePre2011 = Objects.toString(api.getEerCodePre2011(), "");
        this.eerName = Objects.toString(api.getEerName(), "");
        this.gorCode = Objects.toString(api.getGorCode(), "");
        this.gorCodePre2011 = Objects.toString(api.getGorCodePre2011(), "");
        this.gorName = Objects.toString(api.getGorName(), "");
        this.leaCode = Objects.toString(api.getLeaCode(), "");
        this.leaName = Objects.toString(api.getLeaName(), "");
        this.laCode = Objects.toString(api.getLaCode(), "");
        this.laCodePre2011 = Objects.toString(api.getLaCodePre2011(), "");
        this.laName = Objects.toString(api.getLaName(), "");
        this.wardCode = Objects.toString(api.getWardCode(), "");
        this.wardCodePre2011 = Objects.toString(api.getWardCodePre2011(), "");
        this.wardName = Objects.toString(api.getWardName(), "");
        this.censOutArea = Objects.toString(api.getCensOutArea(), "");
    }

    // Getters
    public String getEerCode() {
        return eerCode;
    }

    public String getEerCodePre2011() {
        return eerCodePre2011;
    }

    public String getEerName() {
        return eerName;
    }

    public String getGorCode() {
        return gorCode;
    }

    public String getGorCodePre2011() {
        return gorCodePre2011;
    }

    public String getGorName() {
        return gorName;
    }

    public String getLeaCode() {
        return leaCode;
    }

    public String getLeaName() {
        return leaName;
    }

    public String getLaCode() {
        return laCode;
    }

    public String getLaCodePre2011() {
        return laCodePre2011;
    }

    public String getLaName() {
        return laName;
    }

    public String getWardCode() {
        return wardCode;
    }

    public String getWardCodePre2011() {
        return wardCodePre2011;
    }

    public String getWardName() {
        return wardName;
    }

    public String getCensOutArea() {
        return censOutArea;
    }
}